packages/sdk/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@meeco/sdk",
   "description": "A wrapper library around Meeco's various API SDK's and encryption libraries",
-  "version": "4.25.3",
+  "version": "4.26.0",
```

Comment:
I would incrementing the major version due to a breaking change caused by the removal of the keyPair from the IKeyPairDID interface.  [here](https://github.com/Meeco/js-sdk/pull/280/files#diff-c966faef8fa9e0b70de035562514cc21008d59ea3673e53ae024b3a257bcf97dL4). For instance, the portal relies on keyPair property [here](https://bitbucket.org/meeco/meeco-portal/src/4cb29236c0d6e4d9f7cbf62d329fb8089dd9094f/src/modules/credentials/credentialIssuances/api/issueCredential/issueCredential.ts#lines-26).

Reply:
Thank you, I will change that.